purchaseorder/purchaseorder.proto

```diff
@@ -31,11 +31,11 @@ message PurchaseOrderData {
   // ISO currency code 
   string currency = 12;
   // ordering gross amount including tax 
-  int64 order_amount = 13;
+  bytes order_amount = 13;
```

Comment:
out of curiosity, what is the difference between `order-amount` and `gross_amount` ?

Reply:
Not sure of the distinction. Assumed its PO specific nomenclature